homeassistant/components/filter/sensor.py

```diff
@@ -683,9 +700,13 @@ class TimeThrottleFilter(Filter, SensorEntity):
     One sample per time period.
     """
 
-    def __init__(self, window_size: timedelta, precision: int, entity: str) -> None:
+    def __init__(
+        self, window_size: timedelta, entity: str, precision: int | None = None
```

Comment:
Side note: if you add a `*` after `self`, then it is clearer that it is expected to be all passed via keyword arguments, and their order no longer matters, even if the middle one has defaults.
```suggestion
        self, *, window_size: timedelta, entity: str, precision: int | None = None
```

Reply:
If you do change this, then I think you should set all of them the same.